src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -207,7 +215,8 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
           </Grid>
           <Grid item xs={10}>
             <Link component={RouterLink} to={`${routes.PROJECTS}/${wbsPipe(changeRequest.wbsNum)}`}>
-              {wbsPipe(changeRequest.wbsNum)}
+              {wbsPipe(changeRequest.wbsNum)} - {!isProject(changeRequest.wbsNum) ? projectName + ' - ' : ''}
```

Comment:
The projectName is a type of string || undefined, so we wanna check if the projectName variable exists first. Since we know that the project's name will always be displayed on cr that is related to either the work package or project, we can check if projectName exists on the **projectName section** from  **(WBS#) - (Project Name) - (Work Package Name) structure**. Then, we can check if changeRequest.wbsNum represents a work package or not in the **Work Package Name section**.